src/charts/pie.coffee

```diff
@@ -57,8 +57,7 @@ Ember.Charts.PieComponent = Ember.Charts.ChartComponent.extend(
       value: d.value
       percent: d3.round( 100 * d.value / total )
 
-    key = @get 'sortKey'
-    data.sortBy key
+    _.sortBy data, @get('sortKey')
```

Comment:
I think we're trying to avoid lodash dependencies where we don't need them. It's a big library with a lot of overlap with Ember, and often times it doesn't quite play right with Ember objects and other Ember wrappers.

Though it looks a little less clean I'd rather have a `return [] unless data` or similar here.